qcodes/validators/validators.py

```diff
@@ -1155,12 +1161,12 @@ def __repr__(self) -> str:
         return "<Callable>"
 
 
-class Dict(Validator[TDict[Hashable, Any]]):
+class Dict(Validator[typing.Dict[Hashable, Any]]):
```

Comment:
not dict?
```suggestion
class Dict(Validator[dict[Hashable, Any]]):
```

Reply:
Pyupgrade says no. I don't think dict is indexable in this context until we drop 3.8 support. Since this needs to work at runtime